packages/vx-demo/src/components/PackageList.tsx

```diff
@@ -0,0 +1,344 @@
+import React from 'react';
+import { VxPackage } from '../types';
+
+export default function PackageList({
```

Comment:
tried to use the same component for `/docs` and for the package side nav on the `/docs/axis` page